packages/ui-extensions/docs/surfaces/checkout/reference/examples/ui-components/sheet-layout-content.example.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  extension,
+  Sheet,
+  Button,
+  Link,
+  TextBlock,
+  Image,
+  InlineLayout,
+} from '@shopify/ui-extensions/checkout';
+
+export default extension(
+  'purchase.checkout.footer.render-after',
+  (root, {customerPrivacy}) => {
+    customerPrivacy.subscribe(
+      ({shouldShowBanner}) => {
```

Comment:
Same comment regarding subscribing logic where this will happen every-time the customer privacy changes.

Reply:
It depends if there is something that changes the `customerPrivacy` subscribable. If that happens all this logic will run again and this is appending UI components to the root, which can lead to duplicate components. Declaring the components once, then updating the props would be a better approach we should showcase.

Reply:
Yep, sounds good. Totally fine with patching `checkout-web` after 👍.